python-package/lightgbm/plotting.py

```diff
@@ -487,9 +497,20 @@ def create_tree_digraph(booster, tree_index=0, show_info=None, precision=3,
         The index of a target tree to convert.
     show_info : list of strings or None, optional (default=None)
         What information should be shown in nodes.
-        Possible values of list items:
-        'split_gain', 'internal_value', 'internal_count', 'internal_weight',
-        'leaf_count', 'leaf_weight', 'data_percentage'.
+
```

Comment:
I think this blank line is necessary.

**with blank line**

![image](https://user-images.githubusercontent.com/7608904/100832278-25f64000-342d-11eb-903f-570a9c8da4ea.png)

**without blank line**

![image](https://user-images.githubusercontent.com/7608904/100832349-4d4d0d00-342d-11eb-97ac-2cedb80d6dd4.png)